web/components/admin/ApplicantList/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable camelcase */
+
+class ApplicantList extends React.Component {
+  render() {
+    const { applicants, selectedApplicantId, onApplicantClick, className } = this.props;
+    let applicantNumber = 0;
+
+    return (
+      <div className={`flex dir-col wrap ${className}`}>
```

Comment:
![image](https://user-images.githubusercontent.com/21287547/48974201-240b7200-f006-11e8-8c54-d712fb7f94bc.png)
Right now it's wrapping around into multiple columns, can we make it infinite scroll vertically?

Reply:
Yikes looks like I made it a flexbox for no reason

top 10 web developer problems of 2018